translator/translate/otel/receiver/prometheus/translator.go

```diff
@@ -206,12 +205,14 @@ func addDefaultECSRelabelConfigs(scrapeConfigs []*config.ScrapeConfig, conf *con
 		for _, sdConfig := range scrapeConfig.ServiceDiscoveryConfigs {
 			if fileSDConfig, ok := sdConfig.(*file.SDConfig); ok {
 				for _, filePath := range fileSDConfig.Files {
-					if slices.Contains([]string{filePath}, ecsSDFileName) {
-						scrapeConfig.RelabelConfigs = defaultRelabelConfigs
-						break
+					if filePath == ecsSDFileName {
+						// Prepend default configs to existing customer configs
```

Comment:
Nit: Is this prepending here, or just appending customer config to the defaults?